packages/admin/src/FilamentManager.php

```diff
@@ -145,6 +149,12 @@ public function registerTheme(string | Htmlable | null $theme): void
         $this->theme = $theme;
     }
 
+    public function registerThemeUsingVite(string | array $theme, string $buildDirectory = null): void
+    {
+        $this->viteTheme = $theme;
```

Comment:
New properties don't need introducing, we can just call `app(Vite::class)` and pass it to `$this->theme`